modules/Bio/EnsEMBL/Compara/RunnableDB/EpoExtended/ExtendedGenomeAlignment.pm

```diff
@@ -263,7 +254,7 @@ sub _write_gerp_dataflow {
 sub _parse_results {
     my ($self, $tree_file) = @_;
 
-    #Taken from Analysis/Runnable/LowCoverageGenomeAlignment.pm module
+    #Taken from Production/Analysis/ExtendedGenomeAlignment.pm module
```

Comment:
```suggestion
    #Taken from Compara/Production/Analysis/ExtendedGenomeAlignment.pm module
```
Just incase it looks like we're using a module from production's repo - could be a little misleading

Reply:
Full story :white_haired_man: 

The comment was referring to `ensembl-analysis/modules/Bio/EnsEMBL/Analysis/Runnable/LowCoverageGenomeAlignment.pm` as our `ensembl-compara/modules/Bio/EnsEMBL/Compara/RunnableDB/EpoLowCoverage/LowCoverageGenomeAlignment.pm` was created in 2008 and at that time a standard practice was to have pairs of files: the business code in `ensembl-analysis` and the eHive layer in `ensembl-compara` (though this was not always followed).

`ensembl-compara/modules/Bio/EnsEMBL/Compara/Production/Analysis/Runnable/LowCoverageGenomeAlignment.pm` is a copy of the last version of that ensembl-analysis, with all its change history copied across, so that we don't have any dependencies on `ensembl-analysis` any more.